core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.css

```diff
@@ -505,7 +505,7 @@
   }
 }
 
-@media screen and (max-width: 768px) {
+@media screen and (min-width: 300px) and (max-width: 768px) {
```

Comment:
is min-width needed?

Reply:
Text will still overlap below 300px that's why

Reply:
I can remove it if you want

Reply:
Ok, what happens in this case? Could you attach a screenshot below 300px?

Reply:
#### Here, the screen size is less then 300px and "Topic Editor" text is overlaping with "Back to topic Dashboard" icon
![Screenshot](https://user-images.githubusercontent.com/64253632/180697167-2c426df3-9756-4051-8a84-abd53b4a362c.png)

Reply:
Can you fix this overlap by hiding the 'Topic editor' (text in white) and making the 'Back to Topics ....' text smaller?

Reply:
yeah sure . On it

Reply:
Hey, I had already fixed that part. I forget to mention
Here is the screenshot for screen below 300px
### For Below 300px 
![2](https://user-images.githubusercontent.com/64253632/180929117-ce8b8dec-f2f6-4a4e-b471-0114263c39b0.png)

---
![3](https://user-images.githubusercontent.com/64253632/180929197-d1e36112-033b-4bc6-8db5-13a97a1bd624.png)

### For Above 300px
![1](https://user-images.githubusercontent.com/64253632/180929276-9c771498-2d56-4b39-bf51-abaf841703f0.png)

Reply:
Thanks @Pankaj-SinghR!